topi/python/topi/reduction.py

```diff
@@ -247,16 +266,20 @@ def argmax(data, axis=None, keepdims=False):
         with size one.
         With this option, the result will broadcast correctly against the input array.
 
+    exclude: bool
+        whether to apply rule on axis that are NOT in axis instead
+
     Returns
     -------
     ret : tvm.Tensor
     """
     _argmax = tvm.comm_reducer(fcombine=_argmax_comp, fidentity=_argmax_init, name='argmax')
-    return comm_reduce(data, axis=axis, keepdims=keepdims, func=_argmax, is_idx_reduce=True)
+    return comm_reduce(data, axis=axis, keepdims=keepdims, func=_argmax,
+                       is_idx_reduce=True, exclude=exclude)
 
 
 @tvm.tag_scope(tag=tag.COMM_REDUCE_IDX)
-def argmin(data, axis=None, keepdims=False):
+def argmin(data, axis=None, keepdims=False, exclude=False):
```

Comment:
We should not introduce additional parameters like exclude to break away from numpy compatibility. instead axis is already allowed to be a tuple (Maybe we should think about the same thing for nnvm ops)

Reply:
I agree with you to keep numpy compatible, changed them back.